tensorflow_datasets/testing/test_data/fake_examples/emnist/emnist-test-test-labels-idx1-ubyte

```diff
@@ -0,0 +1 @@
+11111111
```

Comment:
all labels identical ?

Reply:
That dataset included the mnist so i added mnist fake data on there and all data in same on mnist fake data,  is it a problem ?

Reply:
not a huge problem, but I'd prefer more diversity in the test examples.

Reply:
Okay then I will pay attention on my next projects. Thank you !

Reply:
@cyfra testing/mnist.py generates fake input data and it's create 8*1 label. That's why all labels identical.